docs/reference/streamfield/blocks.md

```diff
@@ -339,6 +339,12 @@ All block definitions accept the following optional keyword arguments:
 
     :param required: If true (the default), the field cannot be left blank.
 
+.. autoclass:: wagtail.images.blocks.ImageBlock
+    :show-inheritance:
+
+    An accessibility-focused control to allow the editor to select an existing image, or upload a new one. This has provision for adding alt text, indicating whether images are purely decorative, and is the Wagtail-recommended approach to uploading images. The following additional keyword argument is accepted:
+
+    :param required: If true (the default), the field cannot be left blank.
```

Comment:
`required=False` is documented here, but is not currently implemented - the block always behaves as `required=True`, because the underlying `ImageChooserBlock` is declared as `required=True` and the `clean` method explicitly rejects an empty value.

Reply:
`required=False` now implemented in https://github.com/wagtail/wagtail/pull/12416/commits/01eec77695dad1d0bf7435b62ea1fb83783ff1fa